cdap-ui/templates/common/Table.json

```diff
@@ -4,7 +4,7 @@
     "position": [ "group1" ],
     "group1": {
        "display" : "General",
-       "position" : [ "name", "schema", "schema.row.field" ],
+       "position" : [ "name", "schema", "schema.row.field", "case.sensitive.row.field" ],
```

Comment:
Ya, so not sure if it is required here. Since TableSink needs this property. TableSource doesn't. @bdmogal Can you confirm?